docs/features/pointing_device.md

```diff
@@ -927,3 +928,827 @@ In general the following two functions must be implemented in appropriate locati
+
+## How To Enable
+
+On a keyboard that has a pointing device (_i.e._ `POINTING_DEVICE_ENABLE` _is defined_) pointing modes can be enabled by defining `POINTING_DEVICE_MODES_ENABLE` in `config.h`, defining new modes, and adding mode change keycodes to the keymap.
+
+```c
+// in config.h:
+#define POINTING_DEVICE_MODES_ENABLE
+```
+
+##  Activating Pointing Device Modes
+
+The first 15 pointing device modes can easily be activated by keypress through adding the following keycode macros to a keymap:
+
+#### Built-in keycodes and Keycode Macros (_for _`PM_NONE`_ and the first 16 modes only_)
+
+| Keycode Macro  | Description                                                                                                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `PM_MO(<pm>)`  | Momentary key for pointing mode `<pm>` (i.e active while key pressed deactivate on release)                                                                                                                     |
+| `PM_TG(<pm>)`  | Toggle key for pointing mode `<pm>` (toggle on release, remain until pressed and released again)                                                                                                                |
+| `PM_CYCPRE`    | Increase precision on active device by `POINTING_MODES_PRECISION_STEP` will set to `POINTING_MODES_PRECISION_MIN` if `POINTING_MODES_PRECISION_MAX` is exceeded, (_behaviour reverses if shift is held_)        |
+| `PM_CYCDEV`    | Cycle through devices controlled by pointing modes in order of increasing device_id (_Requires more than one device to be controlled by pointing modes see Pointing Modes for Multiple Pointing Devices below_) |
+
+!> For pointing device modes above mode id 31 a custom keycode would need to be added unless the mode is being activated through some other means (such as on specific layers see (advanced use)[#pointing-modes-advanced-use] below)
```

Comment:
```suggestion
::: warning
For pointing device modes above mode id 31 a custom keycode would need to be added unless the mode is being activated through some other means (such as on specific layers see (advanced use)[#pointing-modes-advanced-use] below)
:::
```

The qmk docs have been ported from `docsify` to `vitepress` in https://github.com/qmk/qmk_firmware/pull/23795/ so some custom markers have changed. See this diff for the hint block changes: https://github.com/qmk/qmk_firmware/pull/23795/files#diff-f6ab771517ff6c0c9fd94867b0f6bb5bd56ee158390ffcd827a558fa25b59ea0

So the documentation needs to be updated for vitepress.